src/ui/pages/Home/index.tsx

```diff
@@ -20,19 +20,19 @@ import CreateIdentityModal from "@src/ui/components/CreateIdentityModal";
 import ConnectionModal from "@src/ui/components/ConnectionModal";
 import Menuable from "@src/ui/components/Menuable";
 import { useAppDispatch } from "@src/ui/ducks/hooks";
-import { useMetaMaskConnect, useMetaMaskWalletInfo } from "@src/ui/services/useMetaMask";
+import { useMetaMaskConnect } from "@src/ui/services/useMetaMask";
 import log from "loglevel";
 
 export default function Home(): ReactElement {
   const dispatch = useAppDispatch();
   const scrollRef = useRef<HTMLDivElement>(null);
   const [fixedTabs, fixTabs] = useState(false);
 
+  useMetaMaskConnect();
```

Comment:
If we want to connect to a wallet, we need to use library https://github.com/Uniswap/web3-react which is responsible for it. If we integrate it will be possible to use eager connect which connects with last provider.

Reply:
Related to #67 so it won't be a problem if we integrate it.